crates/storage/nippy-jar/src/lib.rs

```diff
@@ -485,6 +485,12 @@ impl DataReader {
         let offset_file = File::open(path.as_ref().with_extension(OFFSETS_FILE_EXTENSION))?;
         // SAFETY: File is read-only and its descriptor is kept alive as long as the mmap handle.
         let offset_mmap = unsafe { Mmap::map(&offset_file)? };
+        let offset_size = offset_mmap[0] as u64;
```

Comment:
nit: can we use it down at line 500. doesn't let me make a code suggestion on unchanged lines 🥲

Reply:
yeah ofc i forgor 🤦🏻